subprojects/core/src/main/java/org/gradle/internal/service/scopes/GradleScopeServices.java

```diff
@@ -197,16 +195,4 @@ FileContentCacheFactory createFileContentCacheFactory(
     protected ConfigurationTargetIdentifier createConfigurationTargetIdentifier(GradleInternal gradle) {
         return ConfigurationTargetIdentifier.of(gradle);
     }
-
-    // This needs to go here instead of being “build tree” scoped due to the GradleBuild task.
```

Comment:
❔ Is that not a problem anymore?

Reply:
No, because each build tree now gets the same invocation ID that originates from the action executor.